consensus/core/src/network/mod.rs

```diff
@@ -102,11 +106,18 @@ pub(crate) struct SendBlockResponse {}
 pub(crate) struct FetchBlocksRequest {
     #[prost(bytes = "vec", repeated, tag = "1")]
     block_refs: Vec<Vec<u8>>,
+    // The highest accepted round per authority. The vector represents the round for each authority
+    // and its length should be the same as the committee size.
+    #[prost(uint32, repeated, tag = "2")]
+    highest_accepted_rounds: Vec<Round>,
```

Comment:
We could use a roaring bitmap here, but it might be a little bit of premature optimisation at this point

Reply:
Yeah that would only be useful with byzantine validators.